<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器IP：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器Port：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Edit_serve_IP">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Edit_serve_Port">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_socket_send">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>600</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_serve">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>501</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_socket">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>501</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="But_socket_lianjie">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>596</x>
     <y>420</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>userName:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>596</x>
     <y>470</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>IPAddress:</string>
   </property>
  </widget>
  <widget class="QLabel" name="userLabel">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>420</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>userLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="IPLabel">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>470</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>IPLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>110</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>userName:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>160</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>IPAddress:</string>
   </property>
  </widget>
  <widget class="QLabel" name="userLabel1">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>110</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>userLabel1</string>
   </property>
  </widget>
  <widget class="QLabel" name="IPLabel1">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>160</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>IPLabel1</string>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>226</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>360</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
